src/assessments/visible-focus-order/test-steps/focus-not-obscured.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { generateUID } from 'common/uid-generator';
+import { link } from 'content/link';
+import { title } from 'content/strings/application';
+import * as content from 'content/test/focus/focus-not-obscured';
+import { RestartScanVisualHelperToggle } from 'DetailsView/components/restart-scan-visual-helper-toggle';
+import { FocusAnalyzerConfiguration } from 'injected/analyzers/analyzer';
+import { AnalyzerProvider } from 'injected/analyzers/analyzer-provider';
+import { VisualizationInstanceProcessor } from 'injected/visualization-instance-processor';
+import * as React from 'react';
+import { ManualTestRecordYourResults } from '../../common/manual-test-record-your-results';
+import * as Markup from '../../markup';
+import { Requirement } from '../../types/requirement';
+import { visibleFfocusOrderTestStep } from './test-steps';
```

Comment:
Typo?

```suggestion
import { visibleFocusOrderTestStep } from './test-steps';
```

Reply:
Unfortunately this is a typo that just lives in our repo now. I'm not sure when it was introduced but fixing it isn't part of the work we're doing. It could be fixed in a separate PR but it's not a major priority so it continues to persist.